test/config-next/ra.json

```diff
@@ -44,11 +51,7 @@
     "amqp": {
       "serverURLFile": "test/secrets/amqp_url",
       "insecure": true,
-      "serviceQueue": "RA.server",
-      "SA": {
-        "server": "SA.server",
-        "rpcTimeout": "15s"
-      }
+      "serviceQueue": "RA.server"
```

Comment:
The original gRPC design we came up with said we should keep RPC servers running both a gRPC and AMQP server until we have fully deployed gRPC so clients could speak either protocol and still work. Deleting this config section entirely would cause `boulder-ra` to fail since it needs the AMQP config to run its AMQP RPC server.